README.md

```diff
@@ -1,70 +1,177 @@
+```
+$ git --version
+```
 
-### Analyzing the Bundle Size
+> Likewise for Node installation.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+```
+$ node --version
+```
 
-### Making a Progressive Web App
+### Setup
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+Clone the repository using the GitHub link provided below.
 
-### Advanced Configuration
+### Install
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+In the terminal, go to your file directory and run this command.
 
-### Deployment
+```
+mkdir motorcycle-app
+cd motorcycle-app
+$ git clone https://github.com/jonas-45/motorcycle-booking-react.git 
+```
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+### Usage
 
-### `npm run build` fails to minify
+### Run locally
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+```
+$ cd motorcycle-booking-react
+```
+
+Install the required dependencies to run the project with this command:
+
+```
+$ npm install
+```
+
+Then run it in your browser with this command:
+
+```
+$ npm start -p 3001
+```
+
+Kindly modify the files as needed.
+
+### Run tests
+
+To run tests, please run this command
+
+```
+$ npm test
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Authors <a name="authors"></a>
+
+👤 **Jawad Nazari**
+
+- GitHub: [@Jawad-Nazari](https://github.com/Jawad-Nazari)
+- Twitter: [@Nazari_jawad58](https://twitter.com/Nazari_Jawad58)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/jawad-nazari/)
+
+👤 **Patrick Mukula**
+
+- GitHub: [@Patfarmurs](https://github.com/Patfarmurs)
+- Twitter: [@MukulayengeP](https://twitter.com/MukulayengeP)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/Patrick-Mukula/)
+
+👤  **Wullah Jonas Nuta**
+
+- GitHub: [@jonas-45](https://github.com/jonas-45)
+- Twitter: [@jonaswullah](https://twitter.com/jonaswullah)
+- LinkedIn: [jonas-wullah](https://linkedin.com/in/jonas-wullah)
+
+👤 **Ahmad Saeed**
+
+- Github: [GitHub](https://github.com/ehmaddd/)
+- Twitter: [Twitter](https://twitter.com/ehmaddd_pk)
+- Linkedin: [LinkedIn](https://www.linkedin.com/in/ehmaddd/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭Future Features <a name="future-features"></a>
+
+
```

Comment:
Please , add something for future features like improve style and add more functionality .currently is totally empty.